### DISCUSSION TITLE
Can't find types on initial install

### DISCUSSION THREAD

**Discussion by dvnrsn** [sergiodxa/remix-utils#274] [Q&A]
Cannot find module 'remix-utils/client-only' or its corresponding type declarations.

I'm wondering if this is because of serverModuleFormat: "cjs",

On run dev:
```
Error [ERR_REQUIRE_ESM]: require() of ES Module

Instead change the require of client-only.js to a dynamic import() which is available in all CommonJS modules
```

https://github.com/sergiodxa/remix-utils#usage-with-cjs

in which case I wonder why Indie Stack is CJS.

**Comment by sergiodxa** [comment]
Check that moduleResolution is set as Bundler in your tsconfig.json

**Reply by dvnrsn** [reply]
"moduleResolution": "node",
Interesting, should this be updated in the stacks?

**Reply by sergiodxa** [reply]
The stacks are outdated I think, the default templates are updated.
